hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -386,13 +386,19 @@
     <description>Connection timeout for Ozone client in milliseconds.
     </description>
   </property>
+  <property>
+    <name>ozone.client.stream.buffer.size</name>
+    <value>4MB</value>
+    <tag>OZONE, CLIENT</tag>
+    <description>The size of chunks the client will send to the server.</description>
+  </property>
   <property>
     <name>ozone.client.stream.buffer.flush.size</name>
     <value>16MB</value>
     <tag>OZONE, CLIENT</tag>
     <description>Size which determines at what buffer position , a partial
-      flush will be initiated during write. It should be ideally a multiple
-      of chunkSize.
+      flush will be initiated during write. It should be a multiple
+      of buffer size.
```

Comment:
Can we mention the exact config here instead of buffer size?

Reply:
Sure, will do.